### PR TITLE
VishnuA3_1.html

### PR DESCRIPTION
**Pull request by VIP5614**

### CODE REVIEW

**Review by nikitadamale97 — APPROVED** [review]

**Review by poonam-khode — APPROVED** [review]

File: Vish/assignment-3/Contect.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <style>
+        a{
+            text-decoration: none;
+        }
+    </style>
+</head>
+<body style="background-color: aqua;">
+    <header>
+       <h2 style="text-align: center;">Ambersoft</h2>
+        <center>
```
Comment (Pawan1005) [inline Vish/assignment-3/Contect.html:17]: center tag is outdated

File: Vish/assignment-3/Mi-mean.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <style>
+        a{
+            text-decoration: none;
+        }
+    </style>
+</head>
+<body style="background-color: aqua;">
+   <header>
+       <h2 style="text-align: center;">Ambersoft</h2>
+        <center>
```
Comment (Pawan1005) [inline Vish/assignment-3/Mi-mean.html:17]: same again

File: Vish/assignment-3/VishnuA3_1.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                    </li>
+                    <li>SYSTEM-2
+                        <ul style="list-style-type:disc">
+                            <li>Triveni</li>
+                            <li>Kalyani</li>
+                        </ul>
+                    </li>
+                </ol>
+            </li>
+
```
Comment (Pawan1005) [inline Vish/assignment-3/VishnuA3_1.html:69]: remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
need indentation 
remove extra lines 
add new line at the end of file 
center tag is outdated

**Review by poonam-khode — APPROVED** [review]

**Pull request merged**